packages/flang/build.sh

```diff
@@ -5,13 +5,15 @@ TERMUX_PKG_LICENSE_FILE="flang/LICENSE.TXT"
 TERMUX_PKG_MAINTAINER="@termux"
 LLVM_MAJOR_VERSION=18
 TERMUX_PKG_VERSION=${LLVM_MAJOR_VERSION}.1.7
+TERMUX_PKG_REVISION=1
```

Comment:
Would be worth adding a comment to libllvm build.sh as well, that flang version and revision should be lept in sync, thanks!

Reply:
:+1:, pushed a libllvm comment and version guard check in e801c861b9def151834b2513bc9910022739dea3.